CONTRIBUTING.md

```diff
@@ -0,0 +1,49 @@
+# Contributing to PenText
+
+Thanks for taking the time to contribute!
+
+## Pull Requests
+
+PenText uses the [GitHub flow](https://guides.github.com/introduction/flow/) as
+main versioning workflow. In short,
+
+1. Fork the PenText repository
+2. Create a new branch for each feature, fix, or improvement
+3. Send a pull request from each feature branch to the **master** branch
```

Comment:
may we change the master branch to main?

Reply:
I fully agree that each new *repository* should have its main branch named, well,  main. However, renaming an existing branch might lead to backwards compatibility issues. I see no _pressing_ need myself to rename the current structure. It's just, erm, legacy :)